irohad/pending_txs_storage/impl/pending_txs_storage_impl.cpp

```diff
@@ -0,0 +1,116 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "pending_txs_storage/impl/pending_txs_storage_impl.hpp"
+
+#include "multi_sig_transactions/state/mst_state.hpp"
+
+namespace iroha {
+
+  PendingTransactionStorageImpl::PendingTransactionStorageImpl(
+      StateObservable updatedBatches,
+      BatchObservable preparedBatch,
+      BatchObservable expiredBatch) {
+    updated_batches_subscription_ =
+        updatedBatches.subscribe([this](const SharedState &updatedBatches) {
+          this->updatedBatchesHandler(updatedBatches);
+        });
+    prepared_batch_subscription_ =
+        preparedBatch.subscribe([this](const SharedBatch &preparedBatch) {
+          this->preparedBatchHandler(preparedBatch);
```

Comment:
What is the point to call wrapper instead of `removeBatch` by itself?

Reply:
The wrapper was removed. 

Initially, it was not clear that it will contain the single function call.